pphysx3d/src/scene/mod.rs

```diff
@@ -151,7 +196,7 @@ pub fn narrow_phase(
     for (obj_1, obj_2) in pairs {
         let obj_1 = &objects[*obj_1];
         let obj_2 = &objects[*obj_2];
-        if obj_1.velocity.dot(&obj_2.velocity) <= 0.0 {
+        if obj_1.velocity.dot(&obj_2.velocity) > 0.0 {
```

Comment:
I think it's supposed to be this way, no? The equal sign shouldn't be there anyway since non-moving objects can't collide otherwise.

Reply:
Yes, thank you